.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Issue Template
+about: File an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+This is a suggested issue template for tedana.
+
+If there is other information that would be helpful to include, please don not hesitate to add it!
+
+Before submitting, please check to make sure that the issue is not already addressed.
+If this is a usage question, please checkout NeuroStars here:
```

Comment:
checkout --> check out

Reply:
Good catch ! Another at L13: don not --> do not

Reply:
Both typos should be corrected in 8308d500412cd4a73e58f20147a1c436d7f1f052